### PR TITLE
Update limiter API, examples.

### PR DESCRIPTION
**Pull request by MTCam**
Update limiting API to be compatible with current versions and implementations of limiters from the prediction simulation.
cc: @tulioricci @anderson2981 

**Questions for the review**:
- [ ] Is the scope and purpose of the PR clear?
  - [ ] The PR should have a description.
  - [ ] The PR should have a guide if needed (e.g., an ordering).
- [ ] Is every top-level method and class documented? Are things that should be documented actually so?
- [ ] Is the interface understandable? (I.e. can someone figure out what stuff does?) Is it well-defined?
- [ ] Does the implementation do what the docstring claims?
- [ ] Is everything that is implemented covered by tests?
- [ ] Do you see any immediate risks or performance disadvantages with the design? Example: what do interface normals attach to?

### CODE REVIEW

File: examples/scalar-advdiff.py [hunk 1]
```diff
@@ -150,7 +150,7 @@ def main(actx_class, use_overintegration=False, use_esdg=False,
     else:
         quadrature_tag = None
 
-    def _limit_fluid_cv(cv, pressure, temperature, dd=None):
+    def _limit_fluid_cv(cv, temperature_seed=None, gas_model=None, dd=None):
         # if True:
```
Comment (tulioricci) [inline examples/scalar-advdiff.py:154]: Remove this, please.

File: examples/scalar-advdiff.py [hunk 2]
```diff
@@ -150,7 +150,7 @@ def main(actx_class, use_overintegration=False, use_esdg=False,
...
         #    return cv
```
Comment (tulioricci) [inline examples/scalar-advdiff.py:155]: Remove this comment, please.

**Comment by tulioricci** [comment]
Just running a second check to see if this is really not impacting runtime.

**Review by anderson2981 — APPROVED** [review]
Looks good to me. Aligns with current Y3prediciton main.

**Comment by tulioricci** [comment]
This doesn't seem to affect runtime in a chemistry-complex case. Thanks!

**Pull request merged**